sdl2-ttf.cabal

```diff
@@ -1,39 +1,45 @@
 Cabal-Version:      >= 1.10
 Name:               sdl2-ttf
 Version:            1.0.0
-Maintainer:         Rongcui Dong (karl_1702@188.com)
-Author:             Ömer Sinan Ağacan (omeragacan@gmail.com)
+maintainer:         Rongcui Dong (karl_1702@188.com)
+author:             Ömer Sinan Ağacan (omeragacan@gmail.com)
                   , Sean Chalmers (sclhiannan@gmail.com)
-License-File:       LICENSE
-License:            MIT
-Build-Type:         Simple
-Category:           Foreign binding
-Synopsis:           Binding to libSDL2-ttf
-Description:        Haskell bindings to the sdl2-ttf C++ library <http://www.libsdl.org/projects/SDL_ttf/>.
-Data-files:
+license-file:       LICENSE
+license:            MIT
+build-type:         Simple
+category:           Foreign binding
+synopsis:           Binding to libSDL2-ttf
+description:        Haskell bindings to the sdl2-ttf C++ library <http://www.libsdl.org/projects/SDL_ttf/>.
+data-files:
 extra-source-files: cbits/rendering.h
                     examples/ARIAL.TTF
                     examples/font_test.c
 
-Library
-  Hs-source-dirs:     src
-  Build-Depends:      base >= 3 && < 5, sdl2 >= 2, transformers
+library
+  hs-source-dirs:     src
+  build-depends:      base >= 3 && < 5, sdl2 >= 2, transformers
   default-extensions: ForeignFunctionInterface
-  Exposed-Modules:    SDL.TTF.Types,
+  exposed-modules:    SDL.TTF.Types,
                       SDL.TTF.FFI
                       SDL.TTF
   other-modules:      SDL.TTF.Internals
-  GHC-Options:        -Wall
+  ghc-options:        -Wall
   include-dirs:       cbits
-  C-sources:          cbits/rendering.c
+  c-sources:          cbits/rendering.c
+
+  includes: SDL.h
```

Comment:
I'm not sure. It certainly doesn't hurt things, but you could certainly test it with the omittance of those lines.